### PR TITLE
dwm-status: add xsetroot and alsaUtils as runtime deps

### PR DESCRIPTION
**Pull request by Gerschtli**
###### Motivation for this change

Adds `xorg.xsetroot` and optional `alsaUtils` as runtime dependencies.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Gerschtli** [comment]
@andir cc

File: pkgs/applications/window-managers/dwm/dwm-status.nix [hunk 1]
```diff
@@ -1,4 +1,9 @@
-{ stdenv, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, pkgconfig }:
+{ stdenv, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, pkgconfig, xorg, alsaUtils }:
+
+let
+  runtimeDeps = [ xorg.xsetroot ]
+    ++ lib.optional (alsaUtils != null) alsaUtils;
```
Comment (andir) [inline pkgs/applications/window-managers/dwm/dwm-status.nix]: As you can see in the ofborg failure you are missing an import to the lib attributeset:

```
undefined variable 'lib' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/eval-0-gleber.ewr1.nix.ci/pkgs/applications/window-managers/dwm/dwm-status.nix:5:8
```

File: pkgs/applications/window-managers/dwm/dwm-status.nix [hunk 2]
```diff
@@ -15,11 +20,17 @@ rustPlatform.buildRustPackage rec {
     dbus
     gdk_pixbuf
     libnotify
+    makeWrapper
     pkgconfig
```
Comment (andir) [inline pkgs/applications/window-managers/dwm/dwm-status.nix]: while we are at it lets move `pkgconfig` to `nativeBuildInputs`, also `makeWrapper` should be a `nativeBuildInput`.

File: pkgs/applications/window-managers/dwm/dwm-status.nix [hunk 3]
```diff
@@ -15,11 +20,17 @@ rustPlatform.buildRustPackage rec {
...
   ];
 
   cargoSha256 = "0169k91pb7ipvi0m71cmkppp1klgp5ghampa7x0fxkyrvrf0dvqg";
 
+  postInstall = ''
+    wrapProgram $out/bin/${name} \
```
Comment (andir) [inline pkgs/applications/window-managers/dwm/dwm-status.nix]: Is `${name}` really the right thing to do here?

Is the binary called `dwm-status-0.4.0` or is it called `dwm-status`?

**Review by andir — CHANGES_REQUESTED** [review]

**Comment by andir** [comment]
@grahamcofborg build dwm-status

**Review by andir — APPROVED** [review]

**Pull request merged**